### PR TITLE
Dockerfile: Add checkstyle.jar at build time

### PR DESCRIPTION
**Pull request by fneu**
Closes https://github.com/coala/docker-coala-base/issues/50

### CODE REVIEW

**Review by jayvdb — CHANGES_REQUESTED** [review]
It is /root/.local

**Comment by fneu** [comment]
@jayvdb ofc. changed it. Build passed on dockercloud

File: Dockerfile [hunk 1]
```diff
@@ -305,6 +305,11 @@ RUN curl -L 'http://downloads.sourceforge.net/project/fpgalibre/bakalint/0.4.0/b
   tar xf /tmp/bl.tar.gz -C /root/ && \
   find /tmp -mindepth 1 -prune -exec rm -rf '{}' '+'
 
+# Add checkstyle image
+RUN mkdir -p /root/.local/share/coala-bears/CheckstyleBear && \
+  curl -fsSL https://github.com/coala/bear-runtime-deps/raw/master/CheckstyleBear/checkstyle-6.15-all.jar > /root/.local/share/coala-bears/CheckstyleBear/checkstyle-6.15-all.jar && \
```
Comment (jayvdb) [inline Dockerfile]: Could you use `-o` instead of `>`

**Review by jayvdb — DISMISSED** [review]

File: Dockerfile (hunk 1 above)
Comment (fneu) [reply Dockerfile]: done

**Comment by fneu** [comment]
ack https://github.com/coala/docker-coala-base/pull/131/commits/7f8efe85f187e9734f9d6efa790440b5fe6d6f42

**Comment by fneu** [comment]
@rultor merge

**Pull request merged**